tests/snuba/tsdb/test_tsdb_backend.py

```diff
@@ -35,7 +35,7 @@ def has_shape(data, shape, allow_empty=False):
     Any other object simply has to have the same type.
     If `allow_empty` is set, lists and dicts in `data` will pass even if they are empty.
     """
-    if type(data) != type(shape):
+    if not isinstance(data, type(shape)):
```

Comment:
This one was me, actually. It was needed because we now respond with an `OrderedDict` but have a ton of tests checking against literal `{}`. I agree it's not equivalent but I think it covers the same requirements that `has_shape` is checking for?

Reply:
I think that's OK for this situation, not really considering this one a blocking issue on my side — I've mostly just been trying to tread lightly here.